packages/app-store/routing-forms/pages/route-builder/[...appPages].tsx

```diff
@@ -138,9 +138,14 @@ const Route = ({
       ? eventOptions[0].value.substring(0, eventOptions[0].value.lastIndexOf("/") + 1)
       : "";
 
-  const [customEventTypeSlug, setCustomEventTypeSlug] = useState<string>(
-    !isRouter(route) ? route.action.value.split("/").pop() ?? "" : ""
-  );
+  const [customEventTypeSlug, setCustomEventTypeSlug] = useState<string>("");
+
+  useEffect(() => {
+    if (!isLoading) {
+      const isCustom = !eventOptions.find((eventOption) => eventOption.value === route.action.value);
+      setCustomEventTypeSlug(isCustom ? route.action.value.split("/").pop() ?? "" : "");
```

Comment:
only set the `CustomEventTypeSlug`, when the event type doesn't exist in the `eventOptions`